### PR TITLE
LPS-143532 Only count AccountRole assignees who are active users

### PR DESCRIPTION
**Pull request by RafaelCRC**
@VictorCDP @gislayne-vitorino

### CODE REVIEW

**Comment by RafaelCRC** [comment]
ci:test:sf

**Comment by RafaelCRC** [comment]
ci:test:relevant

**Review by erickmonteiroo — APPROVED** [review]
LGTM!

@drewbrokke Any thoughts about it?

**Comment by erickmonteiroo** [comment]
@RafaelCRC Would you be willing to add test coverage for this? Thanks in advance!

File: modules/apps/account/account-service/src/main/java/com/liferay/account/internal/service/AccountRoleAssigneesRoleLocalServiceWrapper.java [hunk 1]
```diff
@@ -44,8 +46,17 @@ public int getAssigneesTotal(long roleId) throws PortalException {
 					UserGroupRoleTable.INSTANCE.userId
 				).from(
 					UserGroupRoleTable.INSTANCE
+				).leftJoinOn(
```
Comment (drewbrokke) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/internal/service/AccountRoleAssigneesRoleLocalServiceWrapper.java:49]: I think in this case we should use an inner join. 

I don't think it's going to make a practical difference, but it's more expressive of the situation: there should _always_ be a corresponding user for a given entry in the `UserGroupRole` table, so it should always hit an inner join. Again, it's not going to change the result, but someone reading the code should be able to tell that we're not interested in entries that don't also hit the User table.

Thoughts?

File: modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountRoleAssigneesRoleLocalServiceWrapperTest.java [hunk 2]
```diff
@@ -70,6 +73,38 @@ public void testGetAssigneesTotalForAccountSpecificAccountRole()
...
+
+		Assert.assertEquals(
+			2, _roleLocalService.getAssigneesTotal(accountRole.getRoleId()));
+
+		_userLocalService.updateStatus(
+			user2.getUserId(), WorkflowConstants.STATUS_INACTIVE,
+			new ServiceContext());
+
+		Assert.assertEquals(
+			1, _roleLocalService.getAssigneesTotal(accountRole.getRoleId()));
```
Comment (drewbrokke) [inline modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountRoleAssigneesRoleLocalServiceWrapperTest.java:105]: In this case, I think it's sufficient to just append this assertion to the end of the first test case.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Overall, this looks good, thank you @RafaelCRC! Just a couple of comments, nothing huge. Please take a look when you have a moment.

cc @erickmonteiroo

File: modules/apps/account/account-service/src/main/java/com/liferay/account/internal/service/AccountRoleAssigneesRoleLocalServiceWrapper.java (hunk 1 above)
Comment (RafaelCRC) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/internal/service/AccountRoleAssigneesRoleLocalServiceWrapper.java:49]: I agree, i`ll change it. :thumbsup:

**Comment by RafaelCRC** [comment]
ci:test:relevant

**Comment by RafaelCRC** [comment]
ci:test:sf

**Comment by RafaelCRC** [comment]
ci:test:relevant

**Comment by RafaelCRC** [comment]
ci:test:sf

**Review by drewbrokke — APPROVED** [review]
Looks great, thanks @RafaelCRC!

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**